test/WebJobs.Script.Tests/Security/SecretGeneratorTests.cs

```diff
@@ -28,7 +28,9 @@ public void GenerateMasterKeyValue_GeneratesCorrectSecret()
             // "Master01" can be used as the next string literal if this seed
             // is versioned. The bytes are reversed in an attempt to retain a
             // common prefix (left-hand data value) for the the seed if changed.
-            ulong expectedSeed = BitConverter.ToUInt64(Encoding.ASCII.GetBytes("Master00").Reverse().ToArray());
+            byte[] bytes = Encoding.ASCII.GetBytes("Master00");
```

Comment:
Same fixes as I had to make in the other PR. For some reason these tests don't compile locally.